GetDKAN/dkan-tools
# Add dktl-managed assets/sites/default folder

This PR adds commands and functions for creating an assets/sites/default folder in the project root, that is then symlinked into the docroot. It additionally now checks after any command to see if there are any new files in the project root, and tries to `sudo chown` them to the current user if so (we should probably make this behavior optional).

## QA Steps

1. Start a new dktl project. Run `dktl init` in an empty folder. Note that it now creates an assets folder with settings.php files in it.
2. Now run `dktl init --host=acquia`. Most of the init steps should be skipped this time, but you will see a settings.acquia.php file in the assets/sites/default folder. This is just scaffolding; right now it doesn't do anything.
3. Clone DKAN into a `/dkan` folder, run `dktl drupal:make` and then `dktl dkan:install`. The folder should be symlinked into the docroot correctly, and settings files should allow you to run a full install.
4. Run `dktl docker:surl` and open the site in the browser. The site should appear as expected.
